packages/terra-icon/scripts/src/generate-example/index.js

```diff
@@ -1,9 +1,11 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import parseCsv from '../icon-utils/parseCsv';
+import parseCsv from './parseCsv';
```

Comment:
Are the following two files identical? If so, any idea why we have two?

- ../icon-utils/parseCsv
- ./parseCsv

Reply:
They were just slightly different; one used the deprecated part of the csv, the other did not. I went ahead and updated the scripts that relied on them to only use one version, but still produce their original output: [95ea15f](https://github.com/cerner/terra-core/pull/2451/commits/95ea15fb56241f1bdd26df855aee6a5fd8190480)